runtime/compiler/control/CompilationThread.cpp

```diff
@@ -9120,15 +9323,117 @@ TR::CompilationInfo::compilationEnd(J9VMThread * vmThread, TR::IlGeneratorMethod
 
                   if (safeToStore)
                      {
+                     const U_8 *metadataToStore = dataStart;
+                     const U_8 *codedataToStore = codeStart;
+                     int metadataToStoreSize = dataSize;
+                     int codedataToStoreSize = codeSize;
+#ifdef COMPRESS_AOT_DATA
+                     try 
+                        {
+                        if (!comp->getOption(TR_DisableAOTBytesCompression))
+                           {
+                           /**
+                            * For each compiled method data we store in the shareclass cache looks like following.
+                            * -----------------------------------------------------------------------------------------------------
+                            * | CompiledMethodWrapper | J9JITDataCacheHeader | TR_AOTMethodHeader | Metadata | Compiled Code Data |
+                            * -----------------------------------------------------------------------------------------------------
+                            * 
+                            * When we compile a method, we send data from J9JITDataCacheHeader to store in the cache.
+                            * For each methdo Shared Class Cache API adds CompiledMethodWrapper header which holds the size of data in cache, 
```

Comment:
typo: methdo --> method

Reply:
Fixed in https://github.com/eclipse/openj9/pull/5926/commits/66c4368333f9f5ba388b64ec16893d59d9c7eaa1